constants.py

```diff
@@ -1,19 +1,90 @@
-COW_AMM_MAINNET_ADDRESS = "0xBEEf5aFE88eF73337e5070aB2855d37dBF5493A4"
-COW_SETTLEMENT_CONTRACT = "0x9008d19f58aabd9ed0d60971565aa8510560ab41"
+import requests
+from os import getenv
+from dotenv import load_dotenv
+import json
+from web3 import Web3
+
+#it's better to keep token1 to ETH or WETH since it serves as a reference for prices
+API_KEY = "ETHERSCAN_KEY"
+SCAN = "etherscan"
+
+load_dotenv()
+SCAN_API_KEY = getenv(API_KEY)
+INFURA_KEY = getenv("INFURA_KEY")
```

Comment:
The files `constants.py`, `constantsETH.py`, and `constantsGNO.py` look a bit inconsistent.

As an example, this file loads values from a `.env` file, while the other files have it hard coded. This file queries etherscan (not gnosisscan?) other files do not. Maybe some of the code in this file could be moved to a convenience script `scripts/generate_config.py` which gereates a config file with minimal input.